README.md

```diff
@@ -37,7 +37,7 @@ disconnect" logs. You can filter these by exporting the following filter
 string:
 
 ```
-export LOG_FILTER_REGEX="(lost connection after CONNECT from |(dis)?connect from )"
+export LOG_FILTER_REGEX="^.*(connect|disconnect|lost).*$"
```

Comment:
if we only care about if those words are in the sentence, i think the regex could be reduced to 
```suggestion
export LOG_FILTER_REGEX="(connect|disconnect|lost)"
```